tests/phpunit/Presentation/TabsHtmlBuilderUnitTest.php

```diff
@@ -211,7 +194,37 @@ public function testSettingsViewModelContainsValuesWhenUserCanEditConfigurations
 		$this->newTabsHtmlBuilder(
 			config: $config,
 			templateSpy: $templateSpy,
-			user: $this->newUser( canEditConfig: true )
+			enableWikiConfig: false
+		)->createHtml( 'unimportant' );
+
+		$this->assertSame(
+			[],
+			$templateSpy->getArgs()['settings']
+		);
+	}
+
+	public function testSettingsViewModelContainsValuesWhenUserCanEditConfigurations(): void {
+		$config = $this->newConfigFromJson( <<<JSON
+{
+	"itemTypeProperty": "P1337",
+	"configPerItemType": {
+		"Q5976445": {
+			"facets": {
+				"P592": {
+					"type": "list"
+				}
+			}
+		}
+	}
+}
+JSON );
```

Comment:
Continuing chat from https://github.com/ProfessionalWiki/WikibaseFacetedSearch/pull/197#discussion_r1968140892

If this config JSON is here just to avoid the exception, then it is a detail that distracts from what this test is actually doing. Better to hide it behind a method like `$this->newValidConfig()`

Reply:
Actually we already have such a function in the form of `Valid::config();`